src/Init/Data/Nat/Basic.lean

```diff
@@ -112,6 +112,10 @@ attribute [simp] Nat.zero_le
 
 /-! # Helper Bool relation theorems -/
 
+theorem ble_add_one_eq_true :
+    {n m : Nat} → (ble n m) = true → (ble n (m + 1)) = true :=
+  Nat.ble_succ_eq_true
+
```

Comment:
I don't think we should be adding `add_one` analogues of `succ` theorems that are only being used in bootstrapping. I think this is a good example of one. Please be particularly skeptical of making copies of lemmas from Init.Prelude. These will just add pointless API surface.

Reply:
👍 The other lemmas look all very reasonable to me, `Nat.zero_lt_succ` even has a an equivalent `+ 1` version in Mathlib ( [Nat.add_one_pos](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Init/Data/Nat/Basic.html#Nat.add_one_pos))